server/graphql/utils/requestGroup.ts

```diff
@@ -24,7 +26,58 @@ const nextRequestRequestGroupHelper = (requestTypeIds, dataSources): RequestInte
 }
 
 
+const createRequestGroupHelper = async (requestGroup, dataSources, session): Promise<Document> => {
+  const newRequestGroup = await dataSources.requestGroups.create(requestGroup, session)
+
+  newRequestGroup.id = newRequestGroup._id
+  if (requestGroup.requestTypeNames) {
+    const requestTypePromises: Array<Promise<RequestTypeDocument>> = requestGroup.requestTypeNames.map((requestTypeName) => (
+      dataSources.requestTypes.create({ name: requestTypeName, requestGroup: newRequestGroup.id }, session)
+    ))
+
+    await Promise.all(requestTypePromises).then((requestTypes: Array<RequestTypeDocument>) => {
+      newRequestGroup.requestTypes = requestTypes.map((requestType) => requestType._id)
+      return dataSources.requestGroups.update(newRequestGroup, session)
+    })
+  }
+
+  return newRequestGroup
+}
+
 const updateRequestGroupHelper = async (requestGroup, dataSources, session): Promise<Document> => {
+  if (!requestGroup.id) {
+    throw new UserInputError('Missing argument value', { argumentName: 'id' })
+  }
+
+  if (requestGroup.requestTypeNames) {
+    const oldRequestGroup = await dataSources.requestGroups.getById(requestGroup.id)
```

Comment:
I've sometimes encountered issues where nothing gets found if I don't convert the requestGroup id to a string. Not sure if you've encountered something similar if but it works for now i guess it's fine.